src/sentry/static/sentry/app/components/events/interfaces/generic.tsx

```diff
@@ -34,6 +43,10 @@ export default class GenericInterface extends Component {
     };
```

Comment:
Unrelated to the ts conversion, assigning the data prop to state might be unnecessary